### DISCUSSION TITLE
Error on Submit with Notice SubType 1

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#605] [Q&A] [answered]
Hi Lab
I did the first submit with the first notice with sdk 1.8 , of course after that CVS Validation was succeded.
Here the xml 
`<?xml version="1.0" encoding="UTF-8"?>
<PriorInformationNotice xmlns="urn:oasis:names:specification:ubl:schema:xsd:PriorInformationNotice-2" xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2" xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2" xmlns:efac="http://data.europa.eu/p27/eforms-ubl-extension-aggregate-components/1" xmlns:efbc="http://data.europa.eu/p27/eforms-ubl-extension-basic-components/1" xmlns:efext="http://data.europa.eu/p27/eforms-ubl-extensions/1" xmlns:ext="urn:oasis:names:specification:ubl:schema:xsd:CommonExtensionComponents-2">
	<ext:UBLExtensions>
		<ext:UBLExtension>
			<ext:ExtensionContent>
				<efext:EformsExtension>
					<efac:NoticeSubType>
						<cbc:SubTypeCode listName="notice-subtype">1</cbc:SubTypeCode>
					</efac:NoticeSubType>
					<efac:Organizations>
						<efac:Organization>
							<efac:Company>
								<cbc:WebsiteURI>http://bt505organizationcompany.com</cbc:WebsiteURI>
								<cbc:EndpointID>http://bt509organizationcompany.com</cbc:EndpointID>
								<cac:PartyIdentification>
									<cbc:ID schemeName="organization">ORG-0001</cbc:ID>
								</cac:PartyIdentification>
								<cac:PartyName>
									<cbc:Name languageID="ENG">Official name(bt_500_organization_company)</cbc:Name>
								</cac:PartyName>
								<cac:PostalAddress>
									<cbc:StreetName>Postal address(bt_510_a__organization_company)</cbc:StreetName>
									<cbc:AdditionalStreetName>Postal address(bt_510_b__organization_company)</cbc:AdditionalStreetName>
									<cbc:Department>Department(bt_16_organization_company)</cbc:Department>
									<cbc:CityName>Town(bt_513_organization_company)</cbc:CityName>
									<cbc:PostalZone>Postcode(bt_512_organization_company)
									</cbc:PostalZone>
									<cbc:CountrySubentityCode listName="nuts">AL035</cbc:CountrySubentityCode>
									<cac:AddressLine>
										<cbc:Line>Postal address(bt_510_c__organization_company)
										</cbc:Line>
									</cac:AddressLine>
									<cac:Country>
										<cbc:IdentificationCode listName="country">AUT</cbc:IdentificationCode>
									</cac:Country>
								</cac:PostalAddress>
								<cac:PartyLegalEntity>
									<cbc:CompanyID>1914</cbc:CompanyID>
								</cac:PartyLegalEntity>
								<cac:Contact>
									<cbc:Name>Contact point(bt_502_organization_compan</cbc:Name>
									<cbc:Telephone>898989898</cbc:Telephone>
									<cbc:Telefax>898989</cbc:Telefax>
									<cbc:ElectronicMail>bt_506_organization_compan@gmail.com</cbc:ElectronicMail>
								</cac:Contact>
							</efac:Company>
						</efac:Organization>
					</efac:Organizations>
				</efext:EformsExtension>
			</ext:ExtensionContent>
		</ext:UBLExtension>
	</ext:UBLExtensions>
	<cbc:UBLVersionID>2.3</cbc:UBLVersionID>
	<cbc:CustomizationID>eforms-sdk-1.8</cbc:CustomizationID>
	<cbc:ID schemeName="notice-id">f32cb997-0791-4d0c-8be3-a11677a2b9d9</cbc:ID>
	<cbc:IssueDate>2023-07-27+02:00</cbc:IssueDate>
	<cbc:IssueTime>04:43:03+02:00</cbc:IssueTime>
	<cbc:VersionID>01</cbc:VersionID>
	<cbc:RegulatoryDomain>31985R2137</cbc:RegulatoryDomain>
	<cbc:NoticeTypeCode listName="planning">pin-buyer</cbc:NoticeTypeCode>
	<cbc:NoticeLanguageCode listName="language">ENG</cbc:NoticeLanguageCode>
	<cac:ContractingParty>
		<cbc:BuyerProfileURI>http://BT508ProcedureBuyer.com</cbc:BuyerProfileURI>
		<cac:ContractingPartyType>
			<cbc:PartyTypeCode listName="buyer-legal-type">body-pl</cbc:PartyTypeCode>
		</cac:ContractingPartyType>
		<cac:ContractingActivity>
			<cbc:ActivityTypeCode listName="authority-activity">econ-aff</cbc:ActivityTypeCode>
		</cac:ContractingActivity>
		<cac:Party>
			<cac:PartyIdentification>
				<cbc:ID schemeName="organization">ORG-0001</cbc:ID>
			</cac:PartyIdentification>
		</cac:Party>
	</cac:ContractingParty>
	<cac:ContractingParty>
		<cbc:BuyerProfileURI>http://dff.fg</cbc:BuyerProfileURI>
		<cac:ContractingPartyType>
			<cbc:PartyTypeCode listName="buyer-legal-type">body-pl</cbc:PartyTypeCode>
		</cac:ContractingPartyType>
		<cac:ContractingActivity>
			<cbc:ActivityTypeCode listName="authority-activity">education</cbc:ActivityTypeCode>
		</cac:ContractingActivity>
		<cac:Party>
			<cac:PartyIdentification>
				<cbc:ID schemeName="organization">ORG-0001</cbc:ID>
			</cac:PartyIdentification>
		</cac:Party>
	</cac:ContractingParty>
	<cac:TenderingTerms>
		<cac:ProcurementLegislationDocumentReference>
			<cbc:ID>Legal basis of the procedure (ELI - CELEX)(bt_01_c__procedure)</cbc:ID>
			<cbc:DocumentDescription languageID="ENG">Legal basis of the procedure (description)(bt_01_d__procedure)</cbc:DocumentDescription>
		</cac:ProcurementLegislationDocumentReference>
	</cac:TenderingTerms>
	<cac:ProcurementProject>
		<cbc:ID>Internal identifier(bt_22_procedure)</cbc:ID>
		<cbc:Name languageID="ENG">TITOLO PROCEDURA</cbc:Name>
		<cbc:Description languageID="ENG">DESCRIZIONE PROCEDURA</cbc:Description>
		<cbc:ProcurementTypeCode listName="contract-nature">services</cbc:ProcurementTypeCode>
		<cbc:Note languageID="ENG">Additional information(bt_300_procedure)</cbc:Note>
		<cac:ProcurementAdditionalType>
			<cbc:ProcurementTypeCode listName="contract-nature">supplies</cbc:ProcurementTypeCode>
		</cac:ProcurementAdditionalType>
		<cac:MainCommodityClassification>
			<cbc:ItemClassificationCode listName="cpv">75100000</cbc:ItemClassificationCode>
		</cac:MainCommodityClassification>
		<cac:AdditionalCommodityClassification>
			<cbc:ItemClassificationCode listName="cpv">03111100</cbc:ItemClassificationCode>
		</cac:AdditionalCommodityClassification>
		<cac:RealizedLocation>
			<cbc:Description languageID="ENG">Additional information(bt_728_procedure)</cbc:Description>
			<cac:Address>
				<cbc:StreetName>Postal address(bt_5101_a__procedure)</cbc:StreetName>
				<cbc:AdditionalStreetName>Postal address(bt_5101_b__procedure)</cbc:AdditionalStreetName>
				<cbc:CityName>Town(bt_5131_procedure)</cbc:CityName>
				<cbc:PostalZone>Postcode(bt_5121_procedure)</cbc:PostalZone>
				<cbc:CountrySubentityCode listName="nuts">AL011</cbc:CountrySubentityCode>
				<cac:AddressLine>
					<cbc:Line>Postal address(bt_5101_c__procedure)</cbc:Line>
				</cac:AddressLine>
				<cac:Country>
					<cbc:IdentificationCode listName="country">BEL</cbc:IdentificationCode>
				</cac:Country>
			</cac:Address>
		</cac:RealizedLocation>
	</cac:ProcurementProject>
</PriorInformationNotice>`

I got this error :

org.springframework.web.client.HttpClientErrorException$NotFound: 404 Not Found: "{"code":"notice.SDK_NOT_FOUND","message":"No compatible SDK version found for featureVersion=eforms-sdk-1.8","showDetails":false}"

Thanks and Regards
Renato

**Comment by ferrakaop** [comment, marked as answer]
SDK 1.8.0 is not yet fully deployed in our Preview environment. Please bear with us...
